sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -178,13 +178,14 @@ case class DescribeDatabaseCommand(
         Row("Location", CatalogUtils.URIToString(dbMetadata.locationUri)) :: Nil
 
     if (extended) {
-      val properties =
-        if (dbMetadata.properties.isEmpty) {
+      val properties = dbMetadata.properties -- Seq("ownerName", "ownerType")
```

Comment:
why do we hide these 2 properties?

Reply:
They are not tblproperties in hive，maybe we can show as same as hive

Reply:
It's better to know how hive shows them.

Reply:
hive show these in the same level as database name and location,  see https://github.com/apache/hive/blob/fac9ee9099bb4ed8adc921c08e88721f64fd0bd8/ql/src/test/results/clientpositive/describe_database.q.out#L13. We may show them in two different rows after location

Reply:
Shall we do this in another PR or inside this, since I guess it may break many exsit unit tests

Reply:
Shown as hive in columns or just add two more rows,  which introduces user-facing changes, I may need your confirmation @cloud-fan. Or we might start a new jira to track such a change.